students/brgalloway/Lesson_3/mailroom_part_1.py

```diff
@@ -0,0 +1,108 @@
+from operator import itemgetter, attrgetter
+
+# TODO
+# It should have a data structure that holds
+# a list of your donors and a history of the 
+# amounts they have donated. This structure should 
+# be populated at first with at least five 
+# donors, with between 1 and 3 donations each. 
+
+# The script should prompt the user (you) to choose 
+# from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”.
+
+donors_list = [
+    ["Jeff Bezos", 877.33, 1, 877.33],
+    ["Paul Allen", 708.42, 3, 236.14],
+    ["William Gates, III", 653784.49, 2, 326892.24],
+    ["Bill Ackman", 2354.05, 3, 784.68],
+    ["Mark Zuckerberg", 16396.10, 3, 5465.37]  
+]
+
+# Main function to get users input
+def user_prompt(donors_list):
+    display_menu = "Choose one of the following options. \n\n" \
+                "1 - Send a Thank You \n" \
+                "2 - Create a Report \n" \
+                "3 - Quit \n"
+    print(display_menu)
+    prompt = input("Enter a choice to continue: ")   
+    while True:
+        if prompt == "1":
+            find_donor()
+            break
+        elif prompt == "2":
+            generate_report(donors_list)
+            break
+        else: 
+            break
```

Comment:
above main loop has an issue - don't use break in your while True loop, you should continue looping until option "3" is selected.